modules/tinymce/src/themes/silver/main/ts/ui/general/Notification.ts

```diff
@@ -14,7 +14,7 @@ import { Arr, Optional } from '@ephox/katamari';
 
```

Comment:
There's been an icon missed in this file, see line 180 where it uses `getFirst`.

- [ ] Fix missed icon